Vagrantfile

```diff
@@ -57,7 +60,7 @@ if [ "up", "provision" ].include?(ARGV.first)
   install_dependent_roles
 end
 
-ANSIBLE_INVENTORY_PATH = if !ENV["VAGRANT_ENV"].nil? && ENV["VAGRANT_ENV"] == "TEST"
+ANSIBLE_INVENTORY_PATH = if testing?
```

Comment:
We ended up moving the inventories into the `Vagrantfile` to more easily support changing IP addresses: https://github.com/azavea/nyc-trees/commit/bfc76ede45ccaeceb59ef8e06e04ff64c13f75e1.

Not a big deal if you don't see that being a problem down the road though.